Services/sdLitica.AnalyticsManagementCore/DiagnosticsListener.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using sdLitica.Events.Abstractions;
+using sdLitica.Events.Bus;
+using sdLitica.Events.Integration;
+using sdLitica.Relational.Repositories;
+using System;
+
+namespace sdLitica.AnalyticsManagementCore
+{
+    /// <summary>
+    /// Listener for diagnostics info of analytical operations (status, errors etc.)
+    /// </summary>
+    public static class DiagnosticsListener
+    {
+
+        static IEventRegistry _eventRegistry;
+        static IEventBus _eventBus;
+        static OperationRepository _operationRepository;
+        public static IServiceProvider _services;
```

Comment:
'_' means private fields. Please, use a property to external access

Reply:
Thank you for comments. I fixed these problems.